packages/dds/tree/src/edit-manager/editManager.ts

```diff
@@ -241,6 +262,28 @@ export class EditManager<TChangeset, TChangeFamily extends ChangeFamily<any, TCh
 interface Branch<TChangeset> {
     localChanges: Commit<TChangeset>[];
     refSeq: SeqNumber;
+    /**
+     * True iff the commits on the branch were not based on the commit at trunk tip at the time
+     * they were sequenced.
```

Comment:
I find this phrasing a bit confusing. The meaning of "at the time they were sequenced" seems somewhat nebulous to me. While this could be reworded more clearly, I would also consider some alternative definitions. I think these are equivalent to your definition:

A branch is divergent iff it has local changes and there is a change with seqNumber between the branch's refSeq and the seqNumber of the last change in the branch.

In other words, the ref commit followed by the local changes forms a contiguous block in the trunk or final sequence.

Reply:
Regarding "and there is a change with seqNumber between the branch's refSeq and the seqNumber of the last change in the branch": isn't true of all the changes in the branch, not just the last one?

Regarding: "the ref commit followed by the local changes forms a contiguous block in the trunk": did you mean "does not form a contiguous block"?

Reply:
I meant to say:

A branch is divergent iff it has local changes and there is a change _on the trunk but not in the branch_ with seqNumber between the branch's refSeq and the seqNumber of the last change in the branch.

And yes, I dropped a negation at the end.